docs/csharp/programming-guide/xmldoc/exception.md

```diff
@@ -21,22 +21,22 @@ ms.assetid: dd73aac5-3c74-4fcf-9498-f11bff3a2f3c
  cref = " `member`"  
  A reference to an exception that is available from the current compilation environment. The compiler checks that the given exception exists and translates `member` to the canonical element name in the output XML. `member` must appear within double quotation marks (" ").  
   
- For more information on how to create a cref reference to a generic type, see [\<see>](../../../csharp/programming-guide/xmldoc/see.md).  
+ For more information on how to create a cref reference to a generic type, see [Processing the XML File](processing-the-xml-file.md).  
```

Comment:
The problem here, though, is that the target page does not mention crefs; it only mentions member IDs. They're the same, but that requires some clarification here. (for example, "see, which discusses ID strings, which have the same format as crefs.

Reply:
Good catch, I'll reword that